source/neuropod/bindings/java/src/test/java/com/uber/neuropod/NeuropodAdditionTest.java

```diff
@@ -121,75 +120,6 @@ public void loadModel() {
         }
     }
 
-    @Test
-    public void InputDoubleTensor() {
```

Comment:
Why are we removing these tests?

Reply:
I moved (and refactored) all tests that not related to AdditionTest to source/neuropod/bindings/java/src/test/java/com/uber/neuropod/NeuropodAdditionTest.java , you may see it transformed into tensorFromMemory* tests